images/runtime/commonbase/nodeRuntimeBase.Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM oryx-run-base
+
+RUN apt-get update \
+  && apt-get install -y \
+    unzip \
+  && rm -rf /var/lib/apt/lists/*
+  
+# NOTE: This is a list of keys for ALL Node versions and also Yarn package installs.
```

Comment:
This and the Yarn install below should speed up our Node runtime base image builds and also we will be consistent with the Yarn version that we install for all Node runtime versions.

Reply:
cc @arroyc

Reply:
Base images build: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=3245498